MiniApp/Classes/Extensions/MiniApp+WKUserContentController.swift

```diff
@@ -0,0 +1,29 @@
+import WebKit
+
+extension WKUserContentController {
+
+    /// Method to add Mini App custom WKScriptMessageHandler.
+    /// - Parameters:
+    ///   - delegate: Callback protocol to handle the message from the WebView
+    ///   - hostMessageInterface: Message interface protocol of the host application
+    func addMiniAppScriptMessageHandler(delegate: MiniAppCallbackProtocol, hostAppMessageDelegate: MiniAppMessageProtocol) {
+        add(MiniAppScriptMessageHandler(delegate: delegate, hostAppMessageDelegate: hostAppMessageDelegate), name: Constants.javascriptInterfaceName)
+    }
+
+    /// Method to remove the custom WKScriptMessageHandler. Removing the message handler will help to avoid memory leaks
+    func removeMessageHandler() {
```

Comment:
Can the Controller's deinit be extended to automatically remove the messaged handler?

Reply:
@donnie-jp it's a good idea. But it's not really possible here. This expected answer on S.O has a workaround to make the messageHandler being deallocated.

Reply:
Thanks. Ok lets not introduce a hacky workaround and keep the explicit remove 🙂

Reply:
👍